sw-emulator/compliance-test/target-files/model_test.h

```diff
@@ -84,4 +84,4 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #define RVMODEL_CLEAR_MEXT_INT
 
-#endif // _MODEL_TEST_H
\ No newline at end of file
+#endif // _MODEL_TEST_H
```

Comment:
nitpicky: this file shouldn`t be at all touched by this patch

Reply:
Should be fixed (no modification from main) in latest patch.